OutOfSchool/OutOfSchool.WebApi/Services/Database/WorkshopService.cs

```diff
@@ -86,7 +92,8 @@ public async Task<WorkshopCreationResultDto> CreateV2(WorkshopDTO dto)
         _ = dto ?? throw new ArgumentNullException(nameof(dto));
         logger.LogInformation("Workshop creating was started.");
 
-        async Task<(Workshop createdWorkshop, MultipleImageUploadingResult imagesUploadResult, Result<string> coverImageUploadResult)> CreateWorkshopAndDependencies()
+        async Task<(Workshop createdWorkshop, MultipleImageUploadingResult imagesUploadResult, Result<string>
```

Comment:
Try to avoid extra changes where you don't change logic if it wasn't approved before. It will confuse reviewers less while review process